ChessCloudCode/Properties/PublishProfiles/FolderProfile.pubxml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
How does this profile change the instructions in the readme? How about the `build_and_deploy.sh` script here https://github.com/Unity-Technologies/com.unity.services.samples.multiplayer-chess-cloud-code/blob/main/ChessCloudCode/build_and_deploy.sh? Should we remove the script and just update the readme instead?

Reply:
We could leave the script in but just change it to use the new feature